src/initialization/MOM_state_initialization.F90

```diff
@@ -1823,15 +1823,17 @@ subroutine set_Flather_Bdry_Conds(OBC, tv, h, G, PF, tracer_Reg)
     ! Determine where u points are applied at east side 
     do j=jsd,jed ; do I=IsdB,IedB
       if ((I+G%idg_offset) == east_boundary) then !eastern side
-        OBC%OBC_mask_u(I,j) = .true.
-        OBC%OBC_kind_u(I,j) = OBC_FLATHER_E
-        if ((i+1>isd) .and. (i+1<ied) .and. (J>JsdB) .and. (J<JedB)) then
-          OBC%OBC_mask_v(i+1,J) = .true.
-          if (OBC%OBC_kind_v(i+1,J) == OBC_NONE) OBC%OBC_kind_v(i+1,J) = OBC_FLATHER_E
-        endif
-        if ((i+1>isd) .and. (i+1<ied) .and. (J-1>JsdB) .and. (J-1<JedB)) then
-          OBC%OBC_mask_v(i+1,J-1) = .true.
-          if (OBC%OBC_kind_v(i+1,J-1) == OBC_NONE) OBC%OBC_kind_v(i+1,J-1) = OBC_FLATHER_E
+        if (G%mask2dCu(I,j) > 0.99) then
```

Comment:
A minor cleanup request. G%mask2dCu only ever has values of 0.0 or 1.0 . We usually compare to 0. or to 0.5. It is ambiguous which we should use but we prefer that you replace 0.99 with either 0. or 0.5. Thx.

PS> You need only update your user//ksh/open_bc branch to clean this up. The existing PR will automatically use the HEAD of the branch.